src/main/java/uk/gov/hmcts/ccd/domain/service/common/DefaultAuthorisedCaseDataFilter.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.hmcts.ccd.domain.service.common;
```

Comment:
I'm a bit concerned about the number of small security related classes we are introducing recently. Classes that just group operations for convenience and end up to be used only by one client (in this case AuthorisedCaseDetailsSearchOperation).
I'd be tempted to remove this class and move its code and dependencies into AuthorisedCaseDetailsSearchOperation, even thought I agree AuthorisedCaseDetailsSearchOperation would become more complex.
What's your take on this?

Reply:
Well, this class should have been created in the first place and reused everywhere as the grouped code is repeated. That was the intention of creating it. Although, ok to move the logic to AuthorisedCaseDetailsSearchOperation and delete this class.

Reply:
if this code is repeated elsewhere then ok to leave this class, and perhaps in the future we can move some of that code to use this class as well